<launch>
  <include file="$(find simulation_env)/launch/Odometry.launch"/>
  <node pkg="lab2" type="robot_control" name="robot_control" output="screen" clear_params="true">
    <param name="gain_rho" value="0.3"/>
    <param name="gain_alpha" value="1.8"/>
    <param name="gain_beta" value="0.9"/>
  </node>
  <!-- <node pkg="rviz" name="rviz" type="rviz" args="-d $(find lab2)/../haha.rviz"/> -->
</launch>
